app/models/user.rb

```diff
@@ -43,4 +45,16 @@ class User < ApplicationRecord
   def to_full_name
     "#{first_name} #{last_name}"
   end
+
+  def self.extract_user_ids_from_comment(comment)
+    comment.scan(/@\[.*?\]\((\d+)\)/).flatten.map(&:to_i)
+  end
```

Comment:
Since this has to do with a field that is normally located in Response, I wonder if maybe it should go in there? Something like `response.celebrate_user_ids`.